charts/incubator/metabase/values.yaml

```
@@ -0,0 +1,55 @@
+image:
+  repository: tccr.io/truecharts/metabase
+  pullPolicy: IfNotPresent
+  tag: v0.46.7@sha256:96f48cfb662f80278fb4b0356a747eea4d171868c68c7dfec56acc9a24813e96
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 3000
+        port: 13030
+
+persistence:
+  plugins:
+    enabled: true
+    mountPath: "/plugins"
+
+portal:
+  open:
+    enabled: true
+
+workload:
+  main:
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        main:
+          envFrom:
+            - configMapRef:
+                name: metabase-config
+            - secretRef:
+                name: metabase-secrets
+          probes:
+            liveness:
+              type: http
+              path: /
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+            readiness:
+              type: http
+              path: /
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+            startup:
+              type: http
+              path: /
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
```

we can remove the configmap and and secrets as these can be set here instead.

```suggestion
workload:
  main:
    podSpec:
      containers:
        main:
          probes:
            liveness:
              type: http
              path: /
              port: "{{ .Values.service.main.ports.main.targetPort }}"
            readiness:
              type: http
              path: /
              port: "{{ .Values.service.main.ports.main.targetPort }}"
            startup:
              type: http
              path: /
              port: "{{ .Values.service.main.ports.main.targetPort }}"
          env:
            JAVA_TIMEZONE: "{{ .Values.TZ }}"
            MB_DB_TYPE: "postgres"
            MB_DB_USER: "{{ .Values.cnpg.main.user }}"
            MB_DB_DBNAME: "{{ .Values.cnpg.main.database }}"
            MB_DB_PORT: 5432
            MB_DB_HOST:
              secretKeyRef:
                name: cnpg-main-urls
                key: host
            MB_DB_PASS:
              secretKeyRef:
                name: cnpg-main-user
                key: password
```